scripts/util.py

```diff
@@ -51,8 +64,77 @@ def read_source_root_path():
 def read_proof_root_path():
     return read_path_from_stdin("the 'proofs' directory (usually '.')")
 
-def read_litani_path():
-    return read_path_from_stdin("the litani script")
+def is_litani_submodule_in_path(path):
+    try:
+        sys.path.insert(1, path)
+        importlib.import_module("litani")
+        is_litani_submodule_in_path = True
+    except ModuleNotFoundError:
+        print(f"Litani submodule was not found at directory: {path})")
+        is_litani_submodule_in_path = False
+    finally:
+        del sys.path[1]
+    return is_litani_submodule_in_path
```

Comment:
Personal style preference but not blocking. Also you are overwriting the function name with the boolean name `is_litani_submodule_in_path` which probably doesn't cause any issues but is something to be aware of I guess.

```python
    is_in_path = False
    sys.path.insert(1, path)
    try:
        importlib.import_module("litani")
        is_in_path = True
    except ModuleNotFoundError:
        print(f"Litani submodule was not found at directory: {path})")
    del sys.path[1]
    return is_in_path
```

Reply:
Switched to almost this (kept insertion statement inside of try-except block).